v1/transaction/transaction.gunk

```diff
@@ -240,6 +240,9 @@ type CreateTransactionRequest struct {
 
 	// Remarks is an informational note about the transaction.
 	Remarks string `pb:"4" json:"remarks"`
+
+	// TransactionID is the unique identifier of a transaction.
+	TransactionID string `pb:"7" json:"transaction_id"`
```

Comment:
Let's add `SettlementRail` here to allow for multi-rail environments.

Reply:
@Kunde21 What is the data type for **SettlementRail** that we need to add?

Reply:
String type, because the entries will be different in each market.

Reply:
I already add the **SettlementRail** @Kunde21

Reply:
Btw, when we use the SettlementRail @Kunde21 ?

Reply:
There are multiple settlement rails in many markets, main two are realtime vs daily settlement.  They'll have different names in each market.